dist/js/nya-bs-select.js

```diff
@@ -532,7 +532,7 @@ nyaBsSelect.directive('nyaBsSelect', ['$parse', '$document', '$timeout', '$compi
         }
       }
 
-      if(tAttrs.liveSearch === 'true') {
+      if(typeof tAttrs.liveSearch !== 'undefined') {
```

Comment:
This will show the search box no matter what value is passed, including 'false'. Is that your intention here?